src/sentry/static/sentry/app/components/events/errors.jsx

```diff
@@ -44,26 +44,84 @@ class EventErrors extends React.Component {
     const numErrors = errors.length;
     const isOpen = this.state.isOpen;
     return (
-      <EventDataSection event={this.props.event} type="errors" className="errors">
-        <span className="icon icon-alert" />
-        <p>
-          <a className="pull-right errors-toggle" onClick={this.toggle}>
+      <Section>
+        <Summary>
+          <span>
+            <StyledIconWarning color={theme.red} />
```

Comment:
Wouldn't it be better to use the theme inside the `StyledIconWarning` to change the color? then you could get rid of the theme import.